frontend/homework/1/style.css

```diff
@@ -0,0 +1,124 @@
+nav{
+    padding: 10px;
+    background-color: lightpink;
+    list-style-type: none;
+    border: 2px solid red;
+}
+ul{
+    font-style: italic;
+    font-size: larger;
+    background-color: lightpink;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+ul li{
+    margin-top: 10px;
+    float: left;
+    font-size: medium;
+}
+li a{
+    background-color: lightpink;
+    display: block;
+    padding: 10px;
+    margin: 10px;
+    color: green;
+    /* text-decoration: none; */
+}
+
+li a:hover{
+    border: 1px solid black;
+}
+aside{
+    float: right;
+    width: 25%;
+    margin-top: 0px;
+    border: 1px solid brown;
+    justify-content: center;
+    overflow: hidden;
+}
+aside h3{
+    text-align: center;
+    padding: 2px;
+}
+figcaption{
+    text-align: center;
+    background-color: azure;
+    padding: 4px;
+}
+
+article{
+    background-color: rgb(213, 197, 227);
+    overflow: hidden;
+    margin: 10px;
+    padding: 10px;
+    border: 1px solid brown;
+    width: 70%;
+    /* float: left; */
+}
+section{
+    background-color: rgb(213, 197, 227);
+    overflow: hidden;
+    margin: 10px;
+    padding: 10px;
+    border: 1px solid brown;
+    width: 70%;
+    /* float: left; */
+}
+main p{
```

Comment:
Needed to implement the adjacent child selector, adjacent sibling, general sibling, pseudo selectors, elements